deployments/engine/V3/partners/fab/MINTER_DEPLOYMENT_LOGS.log

```diff
@@ -0,0 +1,9 @@
+----------------------------------------
+[INFO] Datetime of minter deployment: 2023-04-13T16:52:58.996Z
+[INFO] Minter deployment configuration file: /Users/jakerockland/Code/artblocks-contracts/deployments/engine/V3/partners/fab/minter-deploy-config.dev.ts
+----------------------------------------
+[INFO] Datetime of minter deployment: 2023-04-13T16:53:13.181Z
+[INFO] Minter deployment configuration file: /Users/jakerockland/Code/artblocks-contracts/deployments/engine/V3/partners/fab/minter-deploy-config.dev.ts
+[INFO] Deploying to network: goerli
+[INFO] Deploying MinterDAExpV4 with deploy args [0x043Eeb8bFd416666b57dd2C5Be439e6fB23e9ce1,0x1A6b570499195139E38Cfa56044Dd748839beE1A]...
+[INFO] MinterDAExpV4 deployed at 0x568D0078D8AFf828d78637209a752f92526150Bf
```

Comment:
hmm,`DEPLOYMENTS.md` should have been updated with a new `# Minter Deployment` section.. did the script have an issue?

Reply:
Note that @ABDruid added this minter in this pr: https://github.com/ArtBlocks/artblocks-contracts/pull/632

Reply:
Ah – yes I think this is because the script failed at the point of allowlisting the minter on the filter, given that filter ACL had already been handed off to partner. 

I think we should probably adjust our logic to try-catch that part, rather than failing from that point on, so that it doesn't interrupt the later logic to updated `DEPLOYMENTS.md`. 

@ryley-o would you have bandwidth to take that on as a quick fix, or should I just add a polish ticket for now?

Reply:
will submit a fix shortly, resolving and merging this PR 🙏

Reply:
AH – fine to merge but going to send a follow up shortly now after actually running the mainnet deploys 😅 

That's my bad, should have added the do-not-merge tag 😄

Reply:
oops 😓